libs/blocks/merch-card/merch-card.js

```diff
@@ -194,6 +195,11 @@ const init = (el) => {
       }
     }
     parseContent(el, altCta, cardType, merchCard);
+    const offerSelection = cardType === 'plans' ? el.querySelector('ul') : null;
```

Comment:
cardtype should be 'product' instead of 'plans', but we have not build that card yet. Just mentioning that we will need to change it in the future